src/main/java/com/gregtechceu/gtceu/client/util/GTFormattingCode.java

```diff
@@ -0,0 +1,88 @@
+package com.gregtechceu.gtceu.client.util;
+
+import com.gregtechceu.gtceu.api.GTValues;
+
+import net.minecraft.ChatFormatting;
+import net.minecraft.util.StringRepresentable;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Arrays;
+
+import static com.gregtechceu.gtceu.api.GTValues.ALL_COLORS;
+import static net.minecraft.ChatFormatting.AQUA;
+import static net.minecraft.ChatFormatting.DARK_GRAY;
+import static net.minecraft.ChatFormatting.GOLD;
+import static net.minecraft.ChatFormatting.GRAY;
+import static net.minecraft.ChatFormatting.RED;
+import static net.minecraft.ChatFormatting.WHITE;
+import static net.minecraft.ChatFormatting.YELLOW;
+
+public enum GTFormattingCode implements StringRepresentable {
```

Comment:
actually, this shouldn't be an enum so that others can create new ones.

Reply:
Okay, I will change it.